### ISSUE TITLE
uninitialized constant Fluent::Input on fluentd 0.14.1

### ISSUE THREAD

**Issue by JoeOrtiz** [repeatedly/fluent-plugin-beats#1]
Getting this new error when upgrading to the new fluentd:

```
 unexpected error error="uninitialized constant Fluent::Input"
 /usr/lib/ruby/gems/2.3.0/gems/fluent-plugin-beats-0.1.0/lib/fluent/plugin/in_beats.rb:19:in `<module:Fluent>'
 /usr/lib/ruby/gems/2.3.0/gems/fluent-plugin-beats-0.1.0/lib/fluent/plugin/in_beats.rb:18:in `<top (required)>'
 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/registry.rb:81:in `block in search'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/registry.rb:79:in `each'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/registry.rb:79:in `search'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/registry.rb:42:in `lookup'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/plugin.rb:146:in `new_impl'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/plugin.rb:100:in `new_input'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/root_agent.rb:229:in `add_source'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/root_agent.rb:95:in `block in configure'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/root_agent.rb:92:in `each'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/root_agent.rb:92:in `configure'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/engine.rb:119:in `configure'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/engine.rb:93:in `run_configure'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/supervisor.rb:594:in `run_configure'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/supervisor.rb:389:in `block in run_worker'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/supervisor.rb:527:in `main_process'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/supervisor.rb:385:in `run_worker'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/lib/fluent/command/fluentd.rb:271:in `<top (required)>'
 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
 /usr/lib/ruby/gems/2.3.0/gems/fluentd-0.14.1/bin/fluentd:5:in `<top (required)>'
 /usr/bin/fluentd:23:in `load'
 /usr/bin/fluentd:23:in `<main>'
```

**Comment by nozino** [comment]
add following line on the top of the file

require 'fluent/input'

**Comment by repeatedly** [comment]
Just fixed and release v0.1.1.
